Small changes

Daminos2836 opened the pull request.

Daminos2836 committed:
Transfer css to scss

Using the scss syntax
Using compass for cross browser compatibility
Minor graphic modification

Daminos2836 committed:
Using sass compile to convert scss to scss

For more information on the modifications: look the scss file

syldium commented:
Marked as draft, requires a *package.json*. May also use Autoprefixer (as most of the JavaScript is recent, the css should work just about everywhere).

Daminos2836 committed:
Delete app.scss

[-] compass removal: unnecessary because js code requiring newer compatibility
[-] scss removing: not use by fkboard's dev

Daminos2836 committed:
Removal sass compile to convert scss to css

[-] scss is not used by fkboard's dev
[*] keeping small changes

Daminos2836 committed:
Use of 4 spaces per tab

[*] Use of 4 spaces for better comparisons with the original code

Daminos2836 commented:
[-] Compass has been removed
[-] The scss code has been deleted because it was not used by fkboard's dev
[\*] The css code has been manually reviewed to remove all traces of autocompilation
[\*] Only small changes remain

syldium reviewed (approved):
Thanks!
